test/passing/tests/doc_comments.mli.ref

```diff
@@ -525,12 +539,34 @@ val k : int
     aliquet at risus sed rhoncus. Ut condimentum rhoncus orci, sit amet
     eleifend erat tempus quis. *)
 
-(** {[ (* a b *) ]} *)
+(** {[
+      (* a b *)
+    ]} *)
 
 val a :
-     fooooooooooooooooooooooooooo (** {[ (* a b *) ]} *)
+     fooooooooooooooooooooooooooo
+     (** {[
+           (* a b *)
+         ]} *)
   -> fooooooooooooooooooooooooo
 
-type x = {a: t  (** {[ (* a b *) ]} *); b: [`A  (** {[ (* a b *) ]} *)]}
-
-type x = A of a  (** {[ (* a b *) ]} *) | B of b  (** {[ (* a b *) ]} *)
+type x =
+  { a: t
+        (** {[
+              (* a b *)
+            ]} *)
```

Comment:
Isn't this comment over indented ?

Reply:
There is a +4 indentation for a comment attached to a record type field (`label_declaration`), the +2 indentation is used when the field breaks on the `:` for example, I'm not a fan of it either but I think there was some backlash the last time I tried to change it.